src/pages/ProxyVotes.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styled from "styled-components";
+
+import IntlMessage from "../components/IntlMessage";
+import { AppState } from "../reducers";
+import {
+  Member,
+  MemberLookupTable,
+  GENESIS_MEMBER
+} from "../reducers/membersNew";
+
+interface OwnProps {}
+interface StateProps {
+  members: Member[];
+}
+interface DispatchProps {}
+interface Props {
+  memberAndVotes: [[Member, number]];
+}
+
+const ProxyVotesElem = styled.main`
+  max-width: 90vw;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
+// TODO these should be nice thumbnails presented in a component also used by Leaderboard.tsx
+const ProxyVotes: React.StatelessComponent<Props> = ({ memberAndVotes }) => {
+  return (
+    <ProxyVotesElem>
+      <section>
+        <IntlMessage id="votes.votes" tagName="h1" />
+        <IntlMessage id="votes.heading" tagName="p" />
+        <ol>
+          {memberAndVotes.map(memberAndVote => {
+            const [member, votes] = memberAndVote;
+            return (
+              <li key={member.uid}>
+                {member.fullName}: {votes}
+              </li>
+            );
+          })}
+        </ol>
+      </section>
+    </ProxyVotesElem>
+  );
+};
+
+function getAncestors(
```

Comment:
P2: I'm fine with all these helpers being here for now since this code doesn't look like it'll necessarily be our long-term way of doing the voting system, but more properly we'd probably have all this analysis happen in `selectors` rather than directly in the component; this component is now far more than just a view, it also contains a great deal of business logic.